go-controller/pkg/kubevirt/router.go

```diff
@@ -150,7 +150,7 @@ func EnsureLocalZonePodAddressesToNodeRoute(watchFactory *factory.WatchFactory,
 			},
 		}
 		if err := libovsdbops.CreateOrReplaceLogicalRouterStaticRouteWithPredicate(nbClient, types.OVNClusterRouter, &ingressRoute, func(item *nbdb.LogicalRouterStaticRoute) bool {
-			matches := item.IPPrefix == ingressRoute.IPPrefix && item.Nexthop == ingressRoute.Nexthop && item.Policy != nil && *item.Policy == *ingressRoute.Policy
+			matches := item.IPPrefix == ingressRoute.IPPrefix && item.Policy != nil && *item.Policy == *ingressRoute.Policy
```

Comment:
I don't understand why removing the `nextHop` from the predicate fixes the issue. 

I (and other reviewers) would appreciate if you could explain that in the PR's description - it currently says:
> This change use just IPPrefix and Nexthop for the predicate point to point routes are updated and not creating preventing from ECMP routes to be configured.

which is wrong IMO - you remove nexthop from the predicate, thus you can't use it for the predicate.

Reply:
Ahh the comment is wrong... I will fix it.

Reply:
Done

Reply:
makes sense. since these are p2p routes - /32 or /128 prefix len routes - we can safely replace them.